### DISCUSSION TITLE
Update request for use with WebUI-Forge and Flux

### DISCUSSION THREAD

**Discussion by rubenmejiac** [adieyal/sd-dynamic-prompts#820] [Ideas]
Kindly requesting updates for the extension for use with WebUI Forge https://github.com/lllyasviel/stable-diffusion-webui-forge and Flux models.

Please consider updating for Forge and more suitable models for Flux-type prompt generation/augmentation.
The extension works in general, but it would be great to have relevant support for new version of Gradio that Forge uses, and also to have at least one newer model for magic prompt and random prompt generation, which use more of a Flux model type of prompting.

Thank you for the extension, it's very useful and feature-rich.

**Comment by hellocatfish** [comment]
agree

**Comment by hellocatfish** [comment]
supported! I installed this extension and restart web UI, and found no "Dynamic Prompts" button.
When I killed the terminal and restart stable-diffusion-webui-forge, "Dynamic Prompts" appeared!

I find it works well, really appreciate the author's work!
![image](https://github.com/user-attachments/assets/f0b33e8b-1ede-4f83-834b-24089b8a3078)
